src/lib/bootloader/grub2_widgets.rb

```diff
@@ -113,30 +113,46 @@ def store
   end
 
   # Represents decision if smt is enabled
-  class Smt < CWM::CheckBox
+  class CpuMitigations < CWM::ComboBox
     include Grub2Widget
 
     def initialize
       textdomain "bootloader"
     end
 
     def label
-      _("Disable Simultaneous &Multithreading")
+      _("CPU Mitigations")
+    end
+
+    def items
+      [
+        [:nosmt, _("Auto + No SMT")],
+        [:auto, _("Keep SMT")],
+        [:off, _("Off")],
+        [:manual, _("Manually")]
```

Comment:
Manual would be more consistent:

```suggestion
        [:manual, _("Manual")]
```